debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/Configurator.java

```diff
@@ -35,23 +35,38 @@
     }
 
     /* package local */ Configurator includeDatabases(String regexList) {
-        return with(MySqlConnectorConfig.DATABASE_WHITELIST, regexList);
+        if (Math.random() >= 0.5) {
```

Comment:
instead of duplicating tests and extending runtimes I decided to randomly choose from the different options. not super nice but will serve the needs imo.

Reply:
Yes, see my other comment elsewhere. We can do this, but it should be logged which one has been chosen.

Reply:
Done